pkg/model/block.go

```diff
@@ -131,3 +131,12 @@ func (blk *Block) String() string {
 
 	return out.String()
 }
+
+func (blk *Block) ManaCost(rmc iotago.Mana) (iotago.Mana, error) {
```

Comment:
I feel like this should be in iota.go

Reply:
I agree, I have added a todo and linked an issue to do this properly when we add scheduler params because it is tricky at the moment anyway and I have added a simplification for now